queue/Gemfile

```diff
@@ -2,6 +2,9 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem "txgh", :path => "../txgh"
```

Comment:
I think there make sense use new hash syntax and single quotes for strings. Just keep everything in one style.